pepys_import/database/default_data/Sensor Types.csv

```diff
@@ -1,3 +1,5 @@
 name
+GPS
+_GPS
```

Comment:
I'm not sure about the history for us having a sensor called `_GPS`.  We shouldn't - we can just use `GPS`.

Reply:
We have it in NMEA and Replay Importer:
https://github.com/debrief/pepys-import/blob/2c8bcaf27ebf4ed91e2862b9609486af4d866f5f/importers/nmea_importer.py#L103

I'll change them as well.

Reply:
Done:
https://github.com/debrief/pepys-import/pull/296/commits/ffded6ce619b0cefa1671471b641785425187d28